cli/central-client/src/test/java/org/ballerinalang/central/client/TestCentralApiClient.java

```diff
@@ -526,4 +527,91 @@ private void setBallerinaHome() {
             System.setProperty(ballerinaInstallDirProp, String.valueOf(Paths.get("build")));
         }
     }
+
+    @Test(description = "Test pull package without Logs", enabled = false)
```

Comment:
We can move these tests to distribution without having it disabled here.

Reply:
Removed disabled tests